OpenOversight/app/models/database.py

```diff
@@ -348,6 +350,21 @@ def __repr__(self):
         return f"<Officer ID {self.id}: {self.full_name()}>"
 
 
+class DecimalAsString(TypeDecorator):
+    cache_ok = True
+    impl = String
+
+    def process_bind_param(self, value, dialect):
+        if value is not None:
+            return str(value)
+        return None
+
+    def process_result_value(self, value, dialect):
+        if value is not None:
+            return Decimal(value)
+        return None
+
```

Comment:
Would this affect how decimals are stored in production? We added an additional check for the sqlite dialect

https://github.com/OrcaCollective/OpenOversight/blob/61e88dae489ee6e2929d01047332427ca4c5f748/OpenOversight/app/models/database.py#L350-L364

Reply:
I didn't see any changes come through when I attempted to create a migration after my change, so I assumed no. If there's another way you could think to validate that assumption, I'm all ears.

I like your solution better though. I think we could also use the config TEST flag instead of the string match. I'll some modifications.

Reply:
With just adding the check for `sqlite` we seem to be getting failures. I'll see if there's another way to do this change.

Reply:
I actually see some changes needed, I'll make the amendment now.